workspace-manager-clienttests/README.md

```diff
@@ -0,0 +1,29 @@
+# workspace-manager-clienttests
+This Gradle project contains Test Runner tests written with the Workspace Manager Service client library.
+
+The Test Runner library [GitHub repository](https://github.com/DataBiosphere/terra-test-runner) has documentation for
+how to write and execute tests.
+
+#### Run a test
+From the workspace-manager-clienttests directory:
+```
+./gradlew  runTest --args="configs/BasicAuthenticated.json /tmp/TR"
```

Comment:
nit: add the `./render-config.sh` call to this code example block. As a reader, I'd expect the first block under "Running a test" to show me the set of commands required to get started.

Reply:
Updated `README.md` with instructions on `render-config.sh`.